# inputs
bool made_contact
---
# outputs
bool is_successful